Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -23,20 +23,24 @@ Release duties have priority over regular squad duties. Please inform your squad
 
 There are usually two release engineers working at any given time. It goes without saying that both engineers need to work together and that constant feedback is vital.
 
-### 2. Release step-by-step
+## 2. Release step-by-step
 
 **Phase 1: Initiation**
 <br/>	*It starts at the end of the sprint (typically when the new sprint starts on Monday)*
 
-1. Cut a release branch:  Create a new branch from develop and push to origin (e.g `release/3.2.0`).
+1. Cut a release branch by creating a new branch from develop and push it to `origin`
+  * For releases concerning both Babylon and Telus at once (most common case), name it `release/{version}` (e.g. `release/3.2.0`)
+  * For releases specific to only Babylon or only Telus, or for releases for other apps (Bupa, NHS111, ...), name it `release/{appname}/{version}` (e.g. `release/bupa/3.2.0`)
 1. Create a slack channel to discuss anything relative to the release (e.g. `ios_release_3_2_0`).
 1. Bump the release version by triggering its command (eg. `/release babylon:3.2.0`) in `#ios-build` (you can run the command every time you want to upload a new build).
-    * This creates a test Tesflight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
+  * This creates a test TestFlight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
 1. Trigger a hockey build from that branch using its command (eg. `/hockeyapp Babylon branch:release/3.17.0`) in `#ios-build`.
```

Comment:
@dmcrodrigues We should actually suggest to do those two actions (TestFlight build and Hockey Build) for both apps (Babylon AND Telus), right?

Reply:
I think @RuiAAPeres and @chitrakotwani are the best persons to answer this because I’m not sure if there’s any coordination/approval with Telus for the releases.

Reply:
@AliSoftware TL; DR - No

Telus branch is cut from Babylon release branch(`release/x.y.z`) only after it is signed off for app-store. This is so that Telus is forked from a fully tested and stable Babylon branch. 

Then,  testing team only tests tickets related to Telus on Telus release branch before signing off Telus build.

Reply:
I think this document can benefit from a section specific to Telus. This knowledge is buried in emails at the moment. Let me do it after your PR is merged.

Reply:
Wait that's not the info I had, and changes the algorithm and the convention quite a bit then 😅

We should then not use a common `release/x.y.z` at all anymore then (and the CRP should not consider creating two CRP tickets from that single branch name) and would instead only use `release/{appname}/{version}` for all cases.

I'll loop back that info in the Google doc since reviewers didn't seem to be aware and that changes the algorithm of we always end up having two branches with different names for Babylon and Telus in all cases, right?

Reply:
Which Google doc are you referring to? 

Telus and Babylon always have different branch names - that's correct

Reply:
Sorry, I'm referring to the Google Doc I mentioned in https://github.com/Babylonpartners/ios-playbook/pull/158#issuecomment-501385077 which I'm working on (and aims to define the the algorithm for the CRP automation while considering all the complexities of our mono-repo-but-multiple-products config)